paywall/src/__tests__/data-iframe/cacheHandler.test.js

```diff
@@ -10,30 +10,39 @@ import {
   setNetwork,
   getFormattedCacheValues,
   setAccountBalance,
+  setKey,
+  getKey,
```

Comment:
the 3 functions below are needed for pulling in all keys and all transaction hashes. Instead of `setKeyIds`, this uses `setLockAddresses` because both locks and keys are indexed by the same value: the lock they are connected to